doc/gallery/introduction/pytensor_intro.ipynb

```diff
@@ -0,0 +1,3407 @@
+{
```

Comment:
<p>The paragraph jump is a bit too abrupt. There's two kind of graphs shown just above (dprint and graphviz) above and nowhere do we mention Apply yet. So is this a third graph? I would perhaps mention the bi-partite thing (and the schematic) only below after we manually walked through the graph?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pytensor/pull/1585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>